modules/ROOT/pages/keycloak/authenticating.adoc

```diff
@@ -34,13 +36,8 @@ Callback authCallback = new Callback<UserPrincipal>() {
 authService.login(options, authCallback);
 ----
 
-Once the browser returns to the app the result can be handled. In the activity provided to the
-`login` method override `onActivityResult`. This handler should always invoke
-`AuthService#handleAuthResponse`, providing the `Intent`. This will exchange the temporary tokens
-returned from `AuthService#login` for long-life tokens and will provide a `UserPrincipal` which can
-be used to access a users details. If this is not invoked you will not have access to the
-`UserPrincipal`.
-
+. Override `handleAuthResult` to handle the result from the browser
```

Comment:
This should be:
In `onActivityResult`, pass the Intent to your instance of `AuthService`.

Reply:
I'll change it to the correct `onActivityResult` but i think we should not describe the intent passing here because it's explained in a note afterwards. Will also remove the comment from the snippet.